core/domain/translatable_object_registry_test.py

```diff
@@ -24,44 +24,47 @@
 class TranslatableObjectRegistryUnitTests(test_utils.GenericTestBase):
     """Test the Registry class in translatable_object_registry."""
 
-    def test_get_object_class_method(self):
+    def test_get_object_class_method(self) -> None:
         """Tests the normal behavior of get_object_class()."""
         retrieved_class = (
             translatable_object_registry.Registry.get_object_class(
                 'TranslatableHtml'))
         self.assertEqual(retrieved_class.__name__, 'TranslatableHtml')
 
-    def test_nontranslatable_class_is_not_gettable(self):
+    def test_nontranslatable_class_is_not_gettable(self) -> None:
         """Tests that trying to retrieve a non-translatable class raises an
         error.
         """
-        with self.assertRaisesRegex(
-            TypeError, 'not a valid translatable object class'):
-            translatable_object_registry.Registry.get_object_class(
+        with self.assertRaisesRegex( # type: ignore[no-untyped-call]
+            TypeError, 'not a valid translatable object class'
+        ):
+            translatable_object_registry.Registry.get_object_class( # type: ignore[call-overload]
```

Comment:
Please add a TODO to get rid of this ignore in the future.

Reply:
ditto below

Reply:
I added the TODO comments for [no-untyped-call] but left others as they are intentionally done to test wrong inputs and will need to be ignored .

Reply:
Ahh i see . Can you also specify me the #issue number that i can add to this TODO comment .

Reply:
ok done @vojtechjelinek PTAL